File: _submissions/18UEE071_Ayush/Day1/highestProduct.cpp
int Solution::maxp3(vector<int> &A) 
{
    sort(A.begin(),A.end());
    int l = A.size();
    int pr1 = A[0]*A[1]*A[l-1];
    int pr2 = A[l-1]*A[l-2]*A[l-3];
    return max(pr1,pr2);
}

File: _submissions/19UCS159_Tharun/Day1/highestProduct.java
public class highestProduct {
  public int maxp3(int[] A) {
    int n = A.length;
    if (n < 3)
      return 0;
    Arrays.sort(A);
    int ans1 = A[n - 1] * A[n - 2] * A[n - 3];
    int ans2 = A[n - 1] * A[0] * A[1];
    return Math.max(ans1, ans2);
  }
}
